liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -810,29 +808,6 @@ protected void doExecute() throws Exception {
         }
     }
 
-    private void processContainerParams() throws MojoExecutionException {
-        if (container) {
-            // this also sets the project property for use in DeployMojoSupport
-            setContainer(true);
-            return;
```

Comment:
I think we still need this part.  So that the Maven -Dcontainer property will cause this to set the project property for use in DeployMojoSupport.